internals/config/default.yml

```diff
@@ -0,0 +1,244 @@
+---
+name: "default name"
+slug: "default"
+
+settings:
+  mobile_phone_verification: false
+  subscriptions: false
+
+# configuration variables for the server app
+server:
+  # put the URL of openwisp-radius here
+  host: "http://127.0.0.1:8000"
+  uuid: organization_uuid
+  secret_key: organization_secret_key
+  timeout: 2 #request timeout period in seconds
+
+# configuration variables for the client
+# Path of all the assets should be defined relative to the organization's folder
+# in assets directory.
+client:
+
+  auto_login: True
+
+  # path of favicon
+  favicon: "favicon.png"
+
+  # path of the custom css file relative to organization's folder in
+  # assets directory.
+  css_path: "index.css"
+  languages:
+    - text: "english"
+      slug: "en"
+  default_language: "en"
+  components:
+    header:
+      logo:
+        url: "openwisp-logo-black.svg" # logo url
+        alternate_text: "openwisp"
+      links:
+        - text:
+            en: "Sign In"
+          url: "/{orgSlug}/login"
+          authenticated: false
+        - text:
+            en: "Sign Up"
+          url: "/{orgSlug}/registration"
+          authenticated: false
+        - text:
+            en: "Change password"
+          url: "/{orgSlug}/change-password"
+          authenticated: true
+        - text:
+            en: "Change phone number"
+          url: "/{orgSlug}/change-phone-number"
+          authenticated: true
+          method: "mobile_phone"
```

Comment:
this link should be removed because it doesn't work when mobile verification is not enabled, but it's shown anyway, which is not good.

I think we could even go as far as have zero links by default.